### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]:Inappropriate control type for the Table in "Web App-Show Properties" tabitem

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1405]
**Repro Steps:**

1. Open and Start Eclipse.
2. Create a new dynamic web project.
3. Navigate to and select Project ->Azure->Publish to web app for containers
4. "Run on web app for containers" dialog will open. 
5. In "Run on web app for containers" dialog, tab to and select  "web app for container" button.
6. Fill the required data and click on "OK" button.
7. Navigate to and select "Show Azure Explorer" button at the toolbar .
8. In "Azure Explorer" , select the created app under web apps and use shift+F10 to open the context menu.
9. Navigate to and select "Show properties" menu item .
10. In "Properties" dialog , tab to table and observe the control type using inspect.

**Actual Result:**
Control type for table is list.
Narrator table commands such as "Caps+F3", "Caps+F8" are working fine.

**Narrator :**
&lt;key&gt;, <say-as interpret-as="spell">1</say-as> of <say-as interpret-as="spell">3</say-as>, selected, 

**NVDA:**
list
grfdgf; Value: <value>  not selected  2 of 2

**Expected Result:**
Control type should be Table.

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![properties-table-inspect](https://user-images.githubusercontent.com/22956130/35685249-2b879ae6-078f-11e8-8290-e72c9ef3ce18.jpg)
![properties-table-narrator](https://user-images.githubusercontent.com/22956130/35685253-2e7b89b0-078f-11e8-9ed3-2297738cb3c9.jpg)
![properties-table-nvda](https://user-images.githubusercontent.com/22956130/35685255-3076f132-078f-11e8-8750-d46fdf8c4616.jpg)

**Comment by jdneo** [comment]
Take the Eclipse official table panel as an example. The Control type is a list.
![untitled](https://user-images.githubusercontent.com/6193897/38671931-5caba4ba-3e7f-11e8-989f-c2f5bd1c7cc9.png)

**Comment by Eskibear** [comment]
https://bugs.eclipse.org/bugs/show_bug.cgi?id=533980

**Comment by Eskibear** [comment]
linked to #1619

**Comment by jdneo** [comment]
* **Importance of experience**: minimal. Very low usage based on telemetry
* **Quality of experience**: not impacted. Users can do the same operations in Azure Portal. 
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.


#### Workaround
The same operations can be achieved in Azure Portal:

![image](https://user-images.githubusercontent.com/6193897/43813175-1131ebe6-9af7-11e8-9982-502d1f07563d.png)

![image](https://user-images.githubusercontent.com/6193897/43813191-24b9bdba-9af7-11e8-9436-0c5d9fa785b9.png)

**Comment by WiproPSAccTest** [comment]
Agreed with the resolution . Can you please remove A11YExclusion, wontfix, Resolved, notreproduce labels wherever applicable and add ExclusionAgreed

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749882
